Resources/Prototypes/DeltaV/Catalog/Fills/Lockers/chiefjustice.yml

```
@@ -0,0 +1,15 @@
+- type: entity
+  parent: LockerChiefJustice
+  id: LockerChiefJusticeFilled
+  name: chief justice's locker
+  components:
+  - type: StorageFill
+    contents:
+    - id: ClothingHeadsetCJ
```

CJ? From GTA San Andreas?

This also doesn't follow the convention in naming for Headsets:
```suggestion
    - id: ClothingHeadsetAltJustice
```